projects/core/src/site-theme/store/actions/site-themes.action.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Action } from '@ngrx/store';
+import { SiteTheme } from '../../../model/misc.model';
+
+export const LOAD_SITE_THEMES = '[Site-theme] Load site themes';
+export const LOAD_SITE_THEMES_FAIL = '[Site-theme] Load site themes Fail';
+export const LOAD_SITE_THEMES_SUCCESS = '[Site-theme] Load site themes Success';
+export const SET_ACTIVE_SITE_THEME = '[Site-theme] Set Active site theme';
+export const SITE_THEME_CHANGE = '[Site-theme] site theme Change';
```

Comment:
`minor:` let's follow the convention and start each word with the capital letter, e.g. `[Site-theme] Load Site Themes`